docs/source/index.rst

```diff
@@ -39,6 +39,7 @@ The :mod:`torchaudio` package consists of I/O, popular datasets and common audio
    compliance.kaldi
    kaldi_io
    utils
+   transducer
```

Comment:
I think this `prototype` is the correct module name.

Reply:
If you mean in terms of compilation, this does compile locally, see preview in description.

If you mean in terms of file organization, [torchtext](https://github.com/pytorch/text/tree/master/docs/source) also groups each prototype/experimental feature in its own doc file.

Is that what you meant?

Reply:
Looking at the `compliance.kaldi` I thought it had to be `prototype.transducer`. If the doc is compiled correctly then it's not an issue.